tools/travis/publish.sh

```diff
@@ -36,6 +36,8 @@ elif [ ${RUNTIME_VERSION} == "5.3" ]; then
   RUNTIME="swift53Action"
 elif [ ${RUNTIME_VERSION} == "5.4" ]; then
   RUNTIME="swift54Action"
+elif [ ${RUNTIME_VERSION} == "5.7" ]; then
```

Comment:
The corresponding change should be made here now: https://github.com/apache/openwhisk-runtime-swift/blob/master/.github/workflows/ci.yaml#L116

Reply:
https://github.com/apache/openwhisk-runtime-swift/pull/153/commits/0707e20ec1070a6231135bf129de81b56bb91f49